AsyncDisplayKit/ASTableView.mm

```diff
@@ -128,6 +128,10 @@ @interface ASTableView () <ASRangeControllerDataSource, ASRangeControllerDelegat
 
   NSIndexPath *_pendingVisibleIndexPath;
 
+  // When we update our data controller in response to an interactive move,
+  // we don't want to tell the table view about the change (it knows!)
+  BOOL _updatingInResponseToInteractiveMove;
```

Comment:
Before, ASTableView had a backdoor into data controller that let it handle interactive moves by blindly modifying the `completedNodes`.

That backdoor is closed now! Instead, table view issues an update through the normal data controller mechanism, and uses an internal flag to say "oh! This update was in response to an interactive move. I don't need to tell `super` about it, because `super` already knows.